package/batocera/core/batocera-configgen/configgen/configgen/utils/virtualmouse.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
```

Comment:
This module should be called `virtual_mouse`. Please also install `ruff` and check this file. There are several issues.